conanfile.py

```diff
@@ -2,25 +2,13 @@
 
 
 class CppStarterProject(ConanFile):
-    # Note: options are copied from CMake boolean options.
-    # When turned off, CMake sometimes passes them as empty strings.
-    options = {
-        "cpp_starter_use_imgui": ["ON", "OFF", ""],
-        "cpp_starter_use_sdl": ["ON", "OFF", ""]
-    }
     name = "CppStarterProject"
     version = "0.1"
     requires = (
         "catch2/2.13.7",
         "docopt.cpp/0.6.2",
         "fmt/8.0.1",
         "spdlog/1.9.2",
```

Comment:
Do we use spdlog ?

Reply:
Great catch. It was used by a target that we don't use. I removed both now.